homeassistant/bootstrap.py

```diff
@@ -395,9 +405,16 @@ def _ensure_loader_prepared(hass: core.HomeAssistant) -> None:
         loader.prepare(hass)
 
 
-def log_exception(ex, domain, config):
+def log_exception(ex, domain, config, hass=None):
     """Generate log exception for config validation."""
     message = 'Invalid config for [{}]: '.format(domain)
+    if hass is not None:
+        _PERSISTENT_VALIDATION.add(domain)
+        message = ('The following platforms contain invalid configuration:  ' +
+                   ', '.join(list(_PERSISTENT_VALIDATION)) +
+                   '  (please check your configuration)')
+        persistent_notification.create(
+            hass, message, 'Invalid config', 'invalid_config')
```

Comment:
Instead of adding many cards with one error each, the message lists them

---

### Invalid platforms
Unable to find the following platforms: sensor.forecast, sensor.loch_ness_monster (please check your configuration)

---